[docs] document manual entity merge failures when sent to PR secondary cluster (#28013)
hashicorp/vault: website/content/partials/known-issues/manual-entity-merge-does-not-persist.mdx

### Manual entity merges sent to a PR secondary cluster are not persisted to storage

#### Affected versions

> - All Enterprise versions from 0.7.0 up

Reviewer: ```suggestion

- Vault Enterprise v0.7.0+
```

#### Issue

> If a manual entity merge is sent to a PR secondary cluster, the change is not persisted to storage, 
> and will be reflected only temporarily on the PR secondary's active node. 

Reviewer: ```suggestion
If you send a manual entity merge to a PR secondary cluster, the change is
temporarily reflected on the active node of the PR secondary cluster, but Vault
does not save the change to persistent storage. 
```

Style correction: write in active voice

#### Workaround

> All manual entity merge requests should be sent to the primary cluster.

Reviewer: ```suggestion

Always send manual entity merge requests to the primary cluster.
```
